runtime/executor/src/main/java/edu/snu/onyx/runtime/executor/data/SerializerManager.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2017 Seoul National University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.snu.onyx.runtime.executor.data;
+
+import edu.snu.onyx.common.coder.Coder;
+import edu.snu.onyx.common.ir.edge.executionproperty.CompressionProperty;
+import edu.snu.onyx.common.ir.executionproperty.ExecutionProperty;
+import edu.snu.onyx.common.ir.executionproperty.ExecutionPropertyMap;
+import edu.snu.onyx.runtime.executor.data.filter.CompressionChainable;
+import edu.snu.onyx.runtime.executor.data.filter.Chainable;
+import edu.snu.onyx.runtime.executor.data.filter.Serializer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+/**
+ * Mapping from RuntimeEdgeId to Coder.
+ */
+public final class SerializerManager {
+  private static final Logger LOG = LoggerFactory.getLogger(SerializerManager.class.getName());
+  private final ConcurrentMap<String, Serializer> runtimeEdgeIdToSerializer = new ConcurrentHashMap<>();
+
+  /**
+   * Constructor.
+   */
+  @Inject
+  public SerializerManager() {
+  }
+
+  /**
+   * Register a coder for runtime edge.
+   *
+   * @param runtimeEdgeId id of the runtime edge.
+   * @param coder         the corresponding coder.
+   * @param propertyMap   ExecutionPropertyMap of runtime edge
+   */
+  public void register(final String runtimeEdgeId,
```

Comment:
Let's just give `RuntimeEdge` instead of these parameters.
Actually, these parameters are almost all of the information that the edge contains.

Reply:
`DataTransferTest`, we cannot get `RuntimeEdge` in `DataTransferTest#setupVertices`, so I left the parameters like above.